react/PrintVisibilityWrapper/PrintVisibilityWrapper.js

```diff
@@ -0,0 +1,41 @@
+import styles from './PrintVisibilityWrapper.less';
+import classNames from 'classnames';
+
+import PropTypes from 'prop-types';
+import React, { PureComponent } from 'react';
+
+export default class PrintVisibilityWrapper extends PureComponent {
+
+  static propTypes = {
+    className: PropTypes.string,
+    children: PropTypes.oneOfType([
+      PropTypes.arrayOf(PropTypes.node),
+      PropTypes.node
+    ]).isRequired,
+    isScreenOnly: PropTypes.bool,
```

Comment:
Can do one required prop aka oneOf ['screenOnly', 'printOnly'], depends on personal and team preferences. Thoughts?